package/MDAnalysis/topology/MMCIFParser.py

```diff
@@ -0,0 +1,331 @@
+"""
```

Comment:
Similar to coordinates/MMCIF.py
- add modelines
- add entry

Reply:
done